tests/CronnerTests/DI/CronnerExtension.phpt

```diff
@@ -42,6 +43,7 @@ class CronnerExtensionTest extends \TestCase
 		);
 		$this->compiler = new Nette\DI\Compiler($builder);
 		$this->compiler->addExtension('cronner', new CronnerExtension());
+		$this->compiler->addExtension('tracy', new TracyExtension());
```

Comment:
This should not happen automatically because I do not use Tracy on some projects where Cronner is used.